static/js/apps/eval_retrieval_generation/overall_feedback.tsx

```diff
@@ -45,80 +44,109 @@ const LOADING_CONTAINER_ID = "form-container";
 // object to hold information about a question
 interface QuestionConfig {
   title: string;
-  question: string;
-  responseOptions: Record<string, string>;
+  questions: {
+    firestoreKey: string;
+    question: string;
+    responseOptions: Record<string, string>;
+  }[];
 }
 
 // Dictionary of feedback stage -> eval type -> question config
 const QUESTION_CONFIG: Record<string, Record<string, QuestionConfig>> = {
   [FeedbackStage.OVERALL_QUESTIONS]: {
     [EvalType.RAG]: {
+      questions: [
+        {
+          firestoreKey: QUERY_OVERALL_QUESTIONS_KEY,
+          question:
+            "Do the generated questions seem sufficient to answer the query?",
+          responseOptions: {
+            [OVERALL_QUESTIONS_OPTION_HAS_MISSING]: "Missing obvious questions",
+            [OVERALL_QUESTIONS_OPTION_NONE_MISSING]:
+              "No obvious questions missing",
+          },
+        },
+      ],
       title: "QUESTIONS EVALUATION",
-      question:
-        "Do the generated questions seem sufficient to answer the query?",
-      responseOptions: {
-        [OVERALL_QUESTIONS_OPTION_HAS_MISSING]: "Missing obvious questions",
-        [OVERALL_QUESTIONS_OPTION_NONE_MISSING]: "No obvious questions missing",
-      },
     },
   },
   [FeedbackStage.OVERALL_ANS]: {
     [EvalType.RIG]: {
+      questions: [
+        {
+          firestoreKey: QUERY_OVERALL_ANS_KEY,
+          question: "How is the overall answer?",
+          responseOptions: {
+            [QUERY_OVERALL_OPTION_HALLUCINATION]: "Found factual inaccuracies",
+            [QUERY_OVERALL_OPTION_OK]: "No obvious factual inaccuracies",
+          },
+        },
+      ],
       title: "OVERALL EVALUATION",
-      question: "How is the overall answer?",
-      responseOptions: {
-        [QUERY_OVERALL_OPTION_HALLUCINATION]: "Found factual inaccuracies",
-        [QUERY_OVERALL_OPTION_OK]: "No obvious factual inaccuracies",
-      },
     },
     [EvalType.RAG]: {
+      questions: [
+        {
+          firestoreKey: QUERY_OVERALL_ANS_KEY,
+          question: "How well does the LLM answer the user query??",
+          responseOptions: {
+            [QUERY_OVERALL_OPTION_IRRELEVANT]: "Does not answer the query",
+            [QUERY_OVERALL_OPTION_PARTIALLY_COMPLETE]:
+              "Partially answers the query",
+            [QUERY_OVERALL_OPTION_OK]: "Fully answers the query",
+          },
```

Comment:
Should we use the existing keys?  Its harder to map across eval results if the keys change, esp since the questions are approx the same as before?

Reply:
used existing keys for the "does not answer" and "fully answers" & added a new key for partial answer

Reply:
My Q was why not use Partially relevant for "partial answer" ?

Reply:
OH, I see what you mean, I misunderstood & thought you wanted to use existing keys from RIG. Updated to the keys we originally created for RAG